Transfer/Sources/TransferAmountCell.swift

```diff
@@ -23,18 +23,24 @@ import Common
 final class TransferAmountCell: UITableViewCell {
     static let reuseIdentifier = "transferAmountCellIdentifier"
     typealias EnteredAmountHandler = (_ value: String) -> Void
+    let decimalSymbol = Character(NumberFormatter().decimalSeparator!)
+    let groupSeparator = Character(NumberFormatter().groupingSeparator!)
 
     var enteredAmountHandler: EnteredAmountHandler?
+    var numberOfDecimals: Int = 0
+    var defaultNumberOfDigits = 0
+    var formattedZeroAmount = "0"
 
-    private lazy var amountTextField: PasteOnlyTextField = {
-        let textField = PasteOnlyTextField(frame: .zero)
+    private lazy var amountTextField: AmountTextField = {
+        let textField = AmountTextField(frame: .zero)
         textField.textAlignment = .center
         textField.keyboardType = UIKeyboardType.decimalPad
         textField.delegate = self
```

Comment:
Add `textField.becomeFirstResponder()` so that on page loads cursor is already focused in the end of amount

Reply:
Ok sure.